src/ArduinoCloudThing.h

```diff
@@ -142,6 +159,7 @@ class ArduinoCloudThing {
   MapParserState handle_Time         (CborValue * value_iter, CborMapData * map_data);
   MapParserState handle_LeaveMap     (CborValue * map_iter, CborValue * value_iter, CborMapData const * const map_data);
 
+  static void   resetMapDataNotBase         (CborMapData * map_data);
```

Comment:
I'd suggest function `resetMapDataNotBase` should better be a member function of the class `CborMapData` - as the `reset` function was.

Reply:
Done